testsuite/documentation/idempotency.md

```diff
@@ -25,9 +25,9 @@ As a standard status, we require that traditional clients and Salt minions have
 If you remove a traditional client or minion, **always re-add** the base channel, otherwise all package and patch tests will fail.
 
 
-## CentOS and Ubuntu minions
+## RedHat-like and Debian-like minions
 
-CentOS and Ubuntu clients are always registered as SSH salt minions by defaut.
+RedHat-like and Debian-like clients are always registered as Salt minions by defaut.
```

Comment:
Why the change here? Are they salt ssh or salt minions? Are we missing ssh minions completely for RHLIKE and DBLIKE? Pretty much same question as above in the table.